Docs/user_guide/modules/terminologies.md

```diff
@@ -0,0 +1,50 @@
+The contexts were created as part of the [QIICR](https://github.com/QIICR) project, in the [dcmqi](https://github.com/QIICR/dcmqi) toolkit.
+
+## Use cases
+
+Define terminology and anatomic region of a data node such as model or markup, or an segment in a segmentation, so that it can be identified unambiguously, and associated with standardized codes when exported to DICOM.
+
+## Panels and their use
+
+In data trees and lists where the color of the objects are shown in a column as a color swatch, then double-clicking the color brings up the terminology selector dialog. These can be found for data nodes in the [Data](data.html), [Models](models.html), [Markups](markups.html) modules, and for the segments within segmentation nodes in the [Segmentations](segmentations.html) and [Segment Editor](segmenteditor.html) modules.
+
+After double-clicking, the basic selector window opens:
+![](https://github.com/Slicer/Slicer/releases/download/docs-resources/terminology_selector_basic.png)
+
+It shows the terminology types in all categories in a searchable list. Custom name and color can be assigned while keeping the selected terminology entry.
+
+Opening up the panes on the left and right, additional options become visible:
+![](https://github.com/Slicer/Slicer/releases/download/docs-resources/terminology_selector_advanced.png)
+
+- Left pane
+  - Category selection can be narrowed (all selected by default).
+  - On the top the terminology context can be changed, as well as new ones loaded from `.json` files.
+- Right pane
+  - Anatomic region can be selected. There are types where anatomic region selection is enabled. These can be types that can be located in multiple part of the body, such as "Blood clot", "Mass", or "Cyst".
+
+## How to
+
+- Find entries
+  - Start typing the name of the category/type/region in the search box above the column
+- Load new terminology/anatomic context
+  - Click the Load button next to the context drop-down and select JSON from local storage
+- Create custom terminology/anatomic context
+  - Start from an existing JSON file, such as the DICOM master list for [terminologies](https://github.com/Slicer/Slicer/blob/master/Modules/Loadable/Terminologies/Resources/SegmentationCategoryTypeModifier-DICOM-Master.json) or [anatomic contexts](https://github.com/Slicer/Slicer/blob/master/Modules/Loadable/Terminologies/Resources/AnatomicRegionAndModifier-DICOM-Master.json). Remove the entries you do not need. Validate the JSON file with the [validator online tool](http://qiicr.org/dcmqi/#/validators).
```

Comment:
I could be wrong, but I believe it should be anatomical over antatomic. I mentioned in other comments similar changes.